### ISSUE TITLE
PREfast check concluded as neutral

### ISSUE THREAD

**Issue by coder137** [microsoft/msvc-code-analysis-action#9]
After the check concluded and the Sarif file is generated, I receive the following message from Github

```
GitHub Code Scanning / PREfast
completed 16 minutes ago in 2s
Error when processing the SARIF file
This check concluded as neutral.
```

## Additional Information

[PR](https://github.com/coder137/build_in_cpp/pull/132)
[Code scanning PREfast](https://github.com/coder137/build_in_cpp/pull/132/checks?check_run_id=4019390911)

**Comment by d-winsor** [comment]
I have validated your analysis completes successfully in a personal fork. You will need to update your workflow to target this fix:

```yml
      - name: Build CMake
        run: cmake --build ${{ env.build }} --config ${{ env.BUILD_TYPE }}

      - name: Run MSVC Code Analysis
        uses: microsoft/msvc-code-analysis-action@main
        # Provide a unique ID to access the sarif output path
        id: run-analysis
        with:
          cmakeBuildDirectory: ${{ env.build }}
          buildConfiguration: ${{ env.BUILD_TYPE }}
          # Ruleset file that will determine what checks will be run
          ruleset: NativeRecommendedRules.ruleset
          ignoredPaths: third_party;build/buildcc/schema/generated
```

**Comment by cschreib** [comment]
I confirm it fixed #10. Thank you!

**Comment by szhorvat** [comment]
@d-winsor I see the same problem here, https://github.com/igraph/igraph/pull/1957/checks?check_run_id=5613512302 Do you have any suggestion on fixing this?
